File: lecture_notes/09 functions/1-function.js
'use strict';

function add1(a, b) {
  return a + b;
}

const add2 = function (a, b) {
  return a + b;
};

const add3 = (a, b) => {
  const res = a + b;
  return res;
};

const add4 = (a, b) => a + b;

console.log({
  add1: add1(10, 20),
  add2: add2(10, 20),
  add3: add3(10, 20),
  add4: add4(10, 20),
});

File: lecture_notes/09 functions/1-function.py
def add1(first_num, sec_num):
    return first_num + sec_num


def add1(n1, n2):
    res = n1 + n2
    return res

add3 = lambda n1, n2: n1 + n2


print(f" {add1(10, 20)}, {add1(7, 20)}")
print(f" {add3(10, 20)}, {add3(7, 20)}")
